repos/system_upgrade/el7toel8/actors/persistentnetnamesconfig/libraries/persistentnetnamesconfig.py

```diff
@@ -64,7 +64,7 @@ def process():
                 )
                 continue
 
-            if rhel7_name != rhel8_name:
+            if rhel7_name != rhel8_name and not os.getenv('LEAPP_NO_NETWORK_RENAMING'):
```

Comment:
Is this gonna work? This actor executes in the `ApplicationPhase` so unless we're forwarding environment variables to the initrd the original environment settings would have been already lost.

Reply:
@shaded-enmity @fernflower 

We can use this one (it is not only about DEVEL env vars nowadays so we may want to change the docstring):

https://github.com/oamg/leapp-repository/blob/22061422b8fc300b4a4c64a1c82b898cd441e6ca/repos/system_upgrade/common/actors/ipuworkflowconfig/libraries/ipuworkflowconfig.py#L45

Reply:
It is then accessed through the Leapp configuration api rather then consuming a message.

E.g.

`env_vars = self.configuration.leapp_env_vars`

Reply:
Aaah. You are right guys. I completely overlooked this. Thanks for another eyes :)

Reply:
Let's also do some checking on the actual value, let's say the variable needs to be set `LEAPP_NO_NETWORK_RENAMING=1` to make it very clear what is the expected outcome. I can imagine people experimenting with various values `LEAPP_NO_NETWORK_RENAMING` and scratching their heads because _any_ value would trigger that code path.

Reply:
ok, so let's do this:
```
if get_env('LEAPP_NO_NETWORK_RENAMING', '0') == '1':
```

Reply:
Done